src/setup_node_env/node_version_validator.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-var */
```

Comment:
Rather than overrides like this, and changes in other files that aren't enforced with eslint but are designed to ensure the node_version check can run before node craps out...

I think we should split this [eslint config override](https://github.com/elastic/kibana/blob/master/.eslintrc.js#L140-L166) into two:

1. files that run before the node_version check and aren't allowed to use things like `let`, `const`, or destructuring syntax
  - scripts/*
  - src/setup_node_env/*

2. files that are not babel-ified but run after the node version check and can use anything except `import`
  - most of the items currently in https://github.com/elastic/kibana/blob/master/.eslintrc.js#L140-L166